src/pages/registration/contributors_tab/components/ContributorRow.tsx

```diff
@@ -156,7 +158,12 @@ export const ContributorRow = ({
             flexDirection: 'column',
             gap: '0.25rem',
             alignItems: 'start',
+            paddingY: '0.5rem',
           }}>
+          <Box sx={{ display: 'flex', gap: '0.3rem', alignItems: 'center' }}>
+            <ErrorIcon color="warning" />
+            <Typography fontWeight={700}>{t('registration.contributors.contributor_is_unidentified')}</Typography>
+          </Box>
```

Comment:
Mangler en sjekk sånn at dette bare vises for uidentifiserte bidragsytere
```suggestion
          {!contributor.identity.id && (
            <Box sx={{ display: 'flex', gap: '0.3rem', alignItems: 'center' }}>
              <ErrorIcon color="warning" />
              <Typography fontWeight={700}>{t('registration.contributors.contributor_is_unidentified')}</Typography>
            </Box>
          )}
```

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/672ffba0-068d-425d-bc90-823671037b78)